packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2074,6 +2093,14 @@ class InputDecoration {
   /// See [Icon], [ImageIcon].
   final Widget suffixIcon;
 
+  /// Optional widget to place on the line after the input.
```

Comment:
It's worth explaining that the suffix widget's baseline will be lined up with the input's baseline.